index.html

```diff
@@ -231,6 +231,14 @@ <h3>Latest news</h3>
         </div>
         <div class="js-home-carousel home-carousel">
           <!-- News box Start -->
+        <div class="news-box">
+        <div class="news-box-title">
+            <span>15/03/2021</span>
+        </div>
+        <div class="news-box-content">
+            <p>Uyuni is participating in <a href="https://developers.google.com/season-of-docs" target="_blank">Google Season of Docs</a>. If you're a technical writer who would like to contribute to the Uyuni documentation, stay tuned for more information.</p>
+        </div>
+        </div>
```

Comment:
Please remove:
```
          <!-- News box Start -->
          <div class="news-box">
            <div class="news-box-title">
              <span>01/03/2021</span>
            </div>
            <div class="news-box-content">
              <p>Uyuni 2021.02 is now available! This mostly a bugfix release including fixes for several <a href="https://saltproject.io/security_announcements/active-saltstack-cve-release-2021-feb-25/" target="_blank">Salt CVES</a> and Ubuntu support for Prometheus exporters' reverse proxy formula. Check the <a href="../pages/stable-version.html#releasenotes">release notes</a> for more details.</p>
            </div>
          </div>
          <!-- News box End -->
```
(251-260). We only keep two news at the same time on this page.

Reply:
Then maybe we shouldn't mention GSoD at all yet? I feel like we don't want to knock the release announce off the front page yet.

Reply:
We should not add GSoD yet because we have not even applied yet. We do not know whether we will be accepted or not.